get-started/site-launch-guide.md

```diff
@@ -1,14 +1,19 @@
 ---
-layout: post
+layout: page
 title: Search Site Launch Guide
 crumbname: Site Launch Guide
 category: manual
 tags: go-live site-launch indexing
+date: April 22, 2019
+last_modified_at: February 3, 2021
 breadcrumbs:
```

Comment:
We should either update the Manual item in these breadcrumbs, or remove the breadcrumbs